zeppelin-server/src/main/java/org/apache/zeppelin/rest/LoginRestApi.java

```diff
@@ -63,53 +178,19 @@ public LoginRestApi() {
   @POST
   @ZeppelinApi
   public Response postLogin(@FormParam("userName") String userName,
-                            @FormParam("password") String password) {
+      @FormParam("password") String password) {
     JsonResponse response = null;
     // ticket set to anonymous for anonymous user. Simplify testing.
     Subject currentUser = org.apache.shiro.SecurityUtils.getSubject();
     if (currentUser.isAuthenticated()) {
       currentUser.logout();
     }
     if (!currentUser.isAuthenticated()) {
-      try {
-        UsernamePasswordToken token = new UsernamePasswordToken(userName, password);
-        //      token.setRememberMe(true);
-
-        currentUser.getSession().stop();
-        currentUser.getSession(true);
-        currentUser.login(token);
-
-        HashSet<String> roles = SecurityUtils.getRoles();
-        String principal = SecurityUtils.getPrincipal();
-        String ticket;
-        if ("anonymous".equals(principal))
-          ticket = "anonymous";
-        else
-          ticket = TicketContainer.instance.getTicket(principal);
 
-        Map<String, String> data = new HashMap<>();
-        data.put("principal", principal);
-        data.put("roles", roles.toString());
-        data.put("ticket", ticket);
-
-        response = new JsonResponse(Response.Status.OK, "", data);
-        //if no exception, that's it, we're done!
-        
-        //set roles for user in NotebookAuthorization module
-        NotebookAuthorization.getInstance().setRoles(principal, roles);
-      } catch (UnknownAccountException uae) {
-        //username wasn't in the system, show them an error message?
-        LOG.error("Exception in login: ", uae);
-      } catch (IncorrectCredentialsException ice) {
-        //password didn't match, try again?
-        LOG.error("Exception in login: ", ice);
-      } catch (LockedAccountException lae) {
-        //account for that username is locked - can't login.  Show them a message?
-        LOG.error("Exception in login: ", lae);
-      } catch (AuthenticationException ae) {
-        //unexpected condition - error?
-        LOG.error("Exception in login: ", ae);
-      }
+      UsernamePasswordToken token = new UsernamePasswordToken(userName, password);
+      //      token.setRememberMe(true);
```

Comment:
this is?

Reply:
Since, we do not have remember me implemented from UI, this has been sitting in comments for a really long time, let me remove it.